SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -37,14 +39,55 @@ def fields(self, m=None):
         f[Srcs, self._solutionType] = u
         return f
 
-    def Jvec(self, m, v, f=None):
+    def getJ(self, m, f=None):
+
+        if self.verbose:
+            print("Calculating J and storing")
+
+        self.model = m
 
         if f is None:
             f = self.fields(m)
 
+        self.Jmat = []
+        AT = self.getA()
+
+        for src in self.survey.srcList:
+            u_src = f[src, self._solutionType]
+            for rx in src.rxList:
+                # wrt f, need possibility wrt m
+                PT = rx.getP(self.mesh, rx.projGLoc(f)).toarray().T
+                df_duTFun = getattr(f, '_{0!s}Deriv'.format(rx.projField),
```

Comment:
for consistency, would you mind using the convention
```python
df_duTFun = getattr(
    f, '_{0!s}Deriv'.format(rx.projField), None
)
```

Reply:
changed.